src/flashbar/flash.tsx

```diff
@@ -170,6 +188,38 @@ export const Flash = React.forwardRef(
       analyticsAttributes[DATA_ATTR_ANALYTICS_SUPPRESS_FLOW_EVENTS] = 'true';
     }
 
+    const getFlashStyles = () => {
```

Comment:
Is this a duplication with `getItemStyles` code from `src/flashbar/collapsible-flashbar.tsx`?

Can we reuse this code somehow?